Decisions.Box/Steps/FileSteps.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Box.V2;
+using Box.V2.Models;
+using DecisionsFramework.Data.DataTypes;
+using DecisionsFramework.Design.Flow;
+using DecisionsFramework.ServiceLayer;
+
+namespace Decisions.Box.Steps;
+
+[AutoRegisterMethodsOnClass(true, "Integration/Box/Files")]
+public class FileSteps
```

Comment:
Do the old steps still exist?

Reply:
I am going to mark them as obsolete and remove the AutoRegister from that class. They will still exist, but they have been replaced by steps that use the Library (new steps).

Reply:
Okay.